ext/ffi/lib.rs

```diff
@@ -2071,32 +2036,59 @@ unsafe extern "C" fn noop_deleter_callback(
 }
 
 #[op(v8)]
-fn op_ffi_get_buf<FP, 'scope>(
+fn op_ffi_get_ptr_value<'scope>(
+  scope: &mut v8::HandleScope<'scope>,
+  state: &mut deno_core::OpState,
+  ptr: serde_v8::Value<'scope>,
+) -> Result<serde_v8::Value<'scope>, AnyError> {
+  check_unstable(state, "Deno.UnsafePointer#getPointerValue");
+
+  let ptr = if let Ok(value) = v8::Local::<v8::External>::try_from(ptr.v8_value)
+  {
+    value.value() as *const c_char
+  } else {
+    return Err(type_error("Invalid FFI pointer value"));
+  };
+
+  let ptr_value = ptr as usize;
+
+  let value: v8::Local<v8::Value> = if ptr as isize > MAX_SAFE_INTEGER {
```

Comment:
Signed integer use implies that the resulting number/pointer may be negative? `ptr as isize < MIN_SAFE_INTEGER` may yield `true`? .. is that intended?

Reply:
`ptr` is usize originally, so it can never be negative. `MAX_SAFE_INTEGER` is `isize` because it happens to be used multiple times to compare against `isize` type values as well.

Reply:
I'm just suggesting that it may make more sense to cast the MAX to usize instead?

..Hypothetically, should a platform ever address the final bit of a pointer, the cast would wrap into negatives. And x86 defines the bits past 48/52/or whatever the range is nowadays, to be filled with ones, so the sjgn bit should be set

Reply:
Aaah, the range past 47 is defined to be filled with copies of bit 47, I think. And 47 is defined to be 0 for user-land pointers and 1 for kernel pointers.

If the top bits were all 1's then our safe number pointers would not work at all and they'd always be BigInts. This is not the case.

But yeah, I'll switch it up to casting into usize... at some point :)